acceptance/bundle/exec/no-auth/script

```diff
@@ -0,0 +1,7 @@
+unset DATABRICKS_HOST
+unset DATABRICKS_TOKEN
+
+# Confirm that bundle exec works for commands that do not require authentication,
```

Comment:
Why do we need this? If people use 'exec' to run script with authentication, is not better to give clear error? If they don't care about authentication, they should not be using "bundle exec" in the first place.

Reply:
It becomes relevant for the script section where we should support commands where authentication is not provided.